backend/main.py

```
@@ -103,65 +148,154 @@ async def google_login(google_token: str):
+
+# Authenticate user from DB
+def authenticate_user(db: Session, username: str, password: str):
+    user = db.query(User).filter(User.username == username).first()
+    if not user or not auth_utils.verify_password(password, user.password):
+        return False
+    return user
+
+
+
+# Get current user from token
+def get_current_user(token: str = Depends(oauth2_scheme), db: Session = Depends(get_db)):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_current_user`

(ANN201)